create-custom-map-style/README.md

```diff
@@ -0,0 +1,990 @@
+	....
+	```
+
+6. Run amplify status to see what will be provisioned
+	```
+	Current Environment: test
+
+	| Category | Resource name                | Operation | Provider plugin   |
+	| -------- | ---------------------------- | --------- | ----------------- |
+	| Auth     | createcustommapstyled00xxx00 | Create    | awscloudformation |
+	| Hosting  | S3AndCloudFront              | Create    | awscloudformation |
+	```
+
+7. Edit `amplify/backend/hosting/S3AndCloudFront/parameters.json` and give the bucket a name you would like:
+	```
+	{
+
+	"bucketName": "createcustommapstyle-<NAME>-hostingbucket"
+
+	}
+	```
+
+8. Push your backend to the cloud `amplify push` - It will create all the resources in the cloud
+	```
+	✔ Successfully pulled backend environment test from the cloud.
+	
+	Current Environment: test
+
+	| Category | Resource name                | Operation | Provider plugin   |
+	| -------- | ---------------------------- | --------- | ----------------- |
+	| Auth     | createcustommapstyled00xxx00 | Create    | awscloudformation |
+	| Hosting  | S3AndCloudFront              | Create    | awscloudformation |
+	
+	? Are you sure you want to continue? Yes
+	⠋ Updating resources in the cloud. This may take a few minutes..
+	```
+
+9. Wait for Cloudformation to provision the necessary resources. Once it's done you should get a message like this:
+	```
+	✔ All resources are updated in the cloud
+	 
+	https://<UNIQUE ID>.cloudfront.net
+	```
+
+10. You might see an *AccessDenied* error when navigating to this URL. You will have to update the hosting bucket policy and specify your Cloudfront Origin Access Identity (OAI) as the  `Principal`  by using its Amazon S3 canonical ID. For example:
+
+	``"Principal": {
+	    "CanonicalUser": "`00x00xx000x000x00x00x0x000xxxxxxxx0x00x00xxxx0x0x0000x0xx00xx0xx`"
+	}``
+
+	Replace  `00x00xx000x000x00x00x0x000xxxxxxxx0x00x00xxxx0x0x0000x0xx00xx0xx`  with the OAI’s canonical ID. You can find the OAI’s canonical ID in the same ways that you find its ID in the Cloudfront management console.
+
+	![Origin Access Identity](media/OAI.png)
+	*Origin Access Identity*
+
+11. The React App can then be run with `npm start` and the deployed app can be updated with `amplify publish` whenever you like. Skip to
+[*Creating Amazon Location Service Resources*](#creating-amazon-location-service-resources) in this README to continue.
+
+## Option 2: Start from Scratch
+### Create a new React app
+1. Install and use Node v12
+	```
+	nvm install 12 
+	nvm use 12
+	```
+2. From your workspace run
+	```
+	npx create-react-app create-custom-map-style 
+	cd create-custom-map-style
+	```
+	This will create a new React desktop application for you
+
+### Amplify Setup
+1. Follow the instructions to initialize a new Amplify project [as outlined here](https://docs.amplify.aws/cli/start/workflows#initialize-new-project)
+2. For the prompts enter the following:
+	 ```
+	amplify init
```

Comment:
```suggestion
	$ amplify init
```

(The `$` is a convention that indicates that the command will be followed by its output.)